SUPPORT.md

```diff
@@ -0,0 +1,11 @@
+# Support
+
+## How to file issues and get help
+
+This project uses GitHub Issues to track bugs and feature requests. Please search the [existing Pylance issues](https://github.com/microsoft/pylance-release/issues) before filing new issues to avoid duplicates. Pylance relies on [pyright](https://github.com/microsoft/pyright) for type checking, so please also reference [existing pyright issues](https://github.com/microsoft/pyright/issues). For new issues, file your bug or feature request as a [new issue](https://github.com/microsoft/pylance-release/issues/new/choose).
```

Comment:
Maybe something like this, to clarify what types of issues might be in pyright's tracker?

```suggestion
This project uses GitHub Issues to track bugs and feature requests. Please search the [existing Pylance issues](https://github.com/microsoft/pylance-release/issues) before filing new issues to avoid duplicates. Pylance relies on [pyright](https://github.com/microsoft/pyright), so for type checking issues please also search for [existing pyright issues](https://github.com/microsoft/pyright/issues). For new issues, file your bug or feature request as a [new issue](https://github.com/microsoft/pylance-release/issues/new/choose).
```